### DISCUSSION TITLE
Types of decorations

### DISCUSSION THREAD

**Discussion by VedankNaik** [iTwin/itwinjs-core#5222] [Q&A]
Hello,

Similar to [glTF decorations](https://www.itwinjs.org/learning/frontend/viewdecorations/#gltf-decorations). Can we import 3D objects from other file formats?

**Comment by pmconne** [comment]
Please be specific about what file formats are of interest to you. glTF is a web standard, so we strive to support it. Do the formats you want provide a conversion to glTF?

**Reply by VedankNaik** [reply]
step, iges, dwg?

**Reply by pmconne** [reply]
Those are not web standards, but you ought to be able to find utilities that can convert them to glTF.
If they contain a lot of geometry, a decorator is probably not the right way to add them to your view. You may want to consider importing them using a connector (e.g., DWG), or publishing them as 3d tilesets.

**Reply by ColinKerr** [reply]
Our connector service supports importing data from step, iges and dwg files.  You can find all of the supported file formats here: https://developer.bentley.com/apis/synchronization/supported-formats/
